### DISCUSSION TITLE
Using Custom Billing Code for all codes not listed

### DISCUSSION THREAD

**Discussion by kgosserand** [CMSgov/price-transparency-guide#419] [Ideas]
We have contractual arrangements where a handful of codes will price with a certain percentage of charge, while other types of codes will price with different percentages of charge.  We would like to use the CSTM-00 reporting type to report the remaining sets of code types outside of this handful of codes. For Example: Provider A has X% of charges for 10 J-codes and Y% of charges for all other CPT/HCPCS codes. We would like to individually report the rate (X%) for the 10 J-codes and report Y% for “all other” CPT/HCPCS codes using the CSTM billing code. 
When using the CSTM-00 reporting type, can we set the text of the negotiated_price object's "description" attribute to further clarify that the set of codes the percentage of charge applies to is the all possible codes for the specified code type not already listed in the file?
 
{
"negotiation_arrangement":  "ffs",
"name": "CPT Codes",
"billing_code_type": "CPT",
"billing_code":  "CSTM-00",
"description":  "All CPT codes not otherwise specified in this file",
"negotiated_rates": [{
   "provider_groups": [{
     "npi": [6666666666],
     "tin":{
       "type": "npi",
       "value": "6666666666"}
   }],
   "negotiated_prices": [{
     "negotiated_type": "percentage",
     "negotiated_rate": Y,
     "expiration_date": "2022-01-01",
     "service_code": ["18", "19", "11"],
     "billing_class": "institutional"}]

**Comment by narenlulla** [comment]
Agree, we also have contracts with waterfall arrangements, fixed price for a set of CPT, then % off Medicare if no Medicare % of charges. CTS-RM for remaing codes would be good to have.

**Comment by DigginForGold** [comment]
Has CMS issued an official decision on this?  We have many contracts that are 99% defined as a percentage of total, but there are a handful of codes (the 1%) on these contracts with specific pricing.  We need an approach to define pricing for everything else not specifically defined at the Plan + Provider Group level.

**Comment by dgolden** [comment]
A broader question is whether there are any plans to update the technical specification? 

We are close to the one-year point, so I would expect CMS to publish a compilation of recommendations, and to solicit industry feedback, a process that should take a month or more.
